msteams-platform/m365-apps/extend-m365-teams-personal-tab.md

```diff
@@ -213,6 +213,16 @@ To view your app in Outlook on the web:
 
     :::image type="content" source="images/outlook-web-more-apps.png" alt-text="The screenshot is an example that shows the ellipses (More apps) option on the side bar of outlook.com to see your installed personal tabs.":::
 
+### Outlook app for Android
+
+To view your app running in Outlook app for Android:
+
+1. Open the Outlook app on your android device and sign in using your developer tenant account. If the Outlook app for Android was already running prior to sideloading your app in Teams, restart Teams to see it in the installed apps section.
```

Comment:
restart Teams? Or restart Outlook in order to see your app in the installed apps section? (Probably the latter)